docprocs/src/test/java/com/yahoo/docprocs/indexing/IndexingProcessorTestCase.java

```diff
@@ -124,33 +82,42 @@ public void testFieldDependingOnTwoInputs() {
             DocumentType inputType = indexer.getDocumentTypeManager().getDocumentType("music");
             DocumentUpdate input = new DocumentUpdate(inputType, "id:ns:music::");
             input.addFieldUpdate(FieldUpdate.createAssign(inputType.getField("artist"), new StringFieldValue("artist1")));
-            // no title
 
             DocumentUpdate output = (DocumentUpdate)process(input);
-            assertEquals(1, output.fieldUpdates().size());
+            assertEquals(2, output.fieldUpdates().size());
             assertAssignment("title", "artist1", output);
+            assertAssignment("combinedWithFallback", "artist1 ", output);
         }
 
         {   // Just title is set
             DocumentType inputType = indexer.getDocumentTypeManager().getDocumentType("music");
             DocumentUpdate input = new DocumentUpdate(inputType, "id:ns:music::");
             input.addFieldUpdate(FieldUpdate.createAssign(inputType.getField("title"), new StringFieldValue("title1")));
-            // no title
 
             DocumentUpdate output = (DocumentUpdate)process(input);
-            assertEquals(1, output.fieldUpdates().size());
+            assertEquals(2, output.fieldUpdates().size());
             assertAssignment("artist", "title1", output);
+            assertAssignment("combinedWithFallback", " title1", output);
         }
-    }
 
-    @Test
-    public void requireThatEmptyDocumentUpdateOutputDoesNotThrow() {
-        DocumentType inputType = indexer.getDocumentTypeManager().getDocumentType("music");
-        DocumentUpdate input = new DocumentUpdate(inputType, "id:ns:music::");
-        Processing proc = new Processing();
-        proc.getDocumentOperations().add(input);
-        indexer.process(proc);
-        assertEquals(0, proc.getDocumentOperations().size());
+        {   // Neither title nor artist is set: Should not update embeddings even though it has fallbacks for all
```

Comment:
We should have a similar set of tests for put operations. For this case I think combinedWithFallback should be set to an empty string if it is a put operation.

Reply:
Yes, done.